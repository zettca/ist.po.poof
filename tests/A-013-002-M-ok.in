1
4
root
6
1
yoda
Jedi
2
0
4
yoda
5
1
0
0
