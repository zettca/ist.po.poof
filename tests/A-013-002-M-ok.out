Menu Principal
1 - Novo
2 - Abrir
0 - Sair
Escolha uma opção: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Login
5 - Menu Shell
6 - Menu Utilizador
0 - Sair
Escolha uma opção: Identificador do utilizador: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Login
5 - Menu Shell
6 - Menu Utilizador
0 - Sair
Escolha uma opção: Menu Utilizador
1 - Criar utilizador
2 - Listar utilizadores
0 - Sair
Escolha uma opção: Identificador do utilizador: Nome do utilizador: Menu Utilizador
1 - Criar utilizador
2 - Listar utilizadores
0 - Sair
Escolha uma opção: root:Super User:/home/root
yoda:Jedi:/home/yoda
Menu Utilizador
1 - Criar utilizador
2 - Listar utilizadores
0 - Sair
Escolha uma opção: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Login
5 - Menu Shell
6 - Menu Utilizador
0 - Sair
Escolha uma opção: Identificador do utilizador: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Login
5 - Menu Shell
6 - Menu Utilizador
0 - Sair
Escolha uma opção: Menu de Shell
1 - Listar
2 - Listar entrada
3 - Remover entrada
4 - Mudar o directório de trabalho
5 - Criar ficheiro
6 - Criar directório
7 - Mostrar o directório de trabalho
8 - Adicionar linha a ficheiro
9 - Ver conteúdo de ficheiro
10 - Mudar permissões de entrada
11 - Mudar dono de entrada
0 - Sair
Escolha uma opção: d - yoda 16 .
d - root 32 ..
Menu de Shell
1 - Listar
2 - Listar entrada
3 - Remover entrada
4 - Mudar o directório de trabalho
5 - Criar ficheiro
6 - Criar directório
7 - Mostrar o directório de trabalho
8 - Adicionar linha a ficheiro
9 - Ver conteúdo de ficheiro
10 - Mudar permissões de entrada
11 - Mudar dono de entrada
0 - Sair
Escolha uma opção: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Login
5 - Menu Shell
6 - Menu Utilizador
0 - Sair
Escolha uma opção: 